web/src/components/Run/Run.tsx

```diff
@@ -0,0 +1,47 @@
+import {useCallback} from 'react';
+import {useStoreActions} from 'react-flow-renderer';
+
+import RunBottomPanel from 'components/RunBottomPanel';
+import {RunLayoutProvider} from 'components/RunLayout';
+import RunTopPanel from 'components/RunTopPanel';
+import {useAppDispatch, useAppSelector} from 'redux/hooks';
+import {setSelectedSpan} from 'redux/slices/TestDefinition.slice';
+import TestDefinitionSelectors from 'selectors/TestDefinition.selectors';
+import {TTest} from 'types/Test.types';
+import {TTestRun} from 'types/TestRun.types';
+
+interface IProps {
+  displayError: boolean;
+  run: TTestRun;
+  test?: TTest;
+}
+
+const Run = ({displayError, run, test}: IProps) => {
```

Comment:
This was our previous `Trace` component. I think it makes sense to start calling it `Run` because this page not only includes the trace. This component now implements the `RunLayoutProvider` and passes the `bottomPanel` and `topPanel` components.